src/tempfile.rs

```diff
@@ -121,7 +121,14 @@ impl TempFile {
 
     /// Returns a reference to the File
     pub fn as_file(&self) -> &File {
-        &self.file
+        // It's safe to unwrap because `file` can be `None` only after calling `into_file`
+        // which consumes this object.
+        self.file.as_ref().unwrap()
+    }
+
+    /// Consumes the TempFile, returning the wrapped file.
```

Comment:
I would add to the doc comment that this also removes the file!

Reply:
Done.